tmuxp.yaml

```diff
@@ -3,6 +3,7 @@ windows:
   - window_name: supervisor
     layout: tiled
     panes:
+      - docker logs $(docker compose ps -q supervisor) && docker attach $(docker compose ps -q supervisor)
       - shell_command:
         - docker exec -it $(docker compose ps -q supervisor) /bin/bash
         - clear && mamba activate modyn && echo "Welcome to the supervisor pane. You have access to the 'modyn-supervisor' command here to submit pipelines."
```

Comment:
I think we might comment this out OR already prepare a pane for the client. This pane will be what the client needs, right now the supervisor server does not need that anymore

Reply:
For this PR, I can make the supervisor window the same as the other servers.

Reply:
done

Reply:
thx